velox/type/DecimalUtil.h

```diff
@@ -28,6 +28,39 @@
 
 namespace facebook::velox {
 
+namespace detail {
+
+/// Represent the varchar fragment.
+///
+/// For example:
+/// | value | wholeDigits | fractionalDigits | exponent | sign |
+/// | 9999999999.99 | 9999999999 | 99 | nullopt | 1 |
+/// | 15 | 15 |  | nullopt | 1 |
+/// | 1.5 | 1 | 5 | nullopt | 1 |
+/// | -1.5 | 1 | 5 | nullopt | -1 |
+/// | 31.523e-2 | 31 | 523 | -2 | 1 |
+struct DecimalComponents {
```

Comment:
Can we move them to DecimalUtil private scope? We should avoid namespace detail if possible.

Reply:
Done, thanks.